lib/cortex/controller.ex

```diff
@@ -7,52 +7,97 @@ defmodule Cortex.Controller do
   alias Cortex.{Reloader,TestRunner}
 
 
+
+  ##########################################
+  # Types
+  ##########################################
+
+  @type focus :: keyword | nil
+
+
+
   ##########################################
   # Public API
   ##########################################
 
+
   def start_link() do
     GenServer.start_link(__MODULE__, [], name: __MODULE__)
   end
 
+
   def pipeline(timeout \\ 5000) do
     GenServer.call(__MODULE__, :get_pipeline, timeout)
   end
 
+
   def run_all, do: GenServer.cast(__MODULE__, :run_all)
 
+
+  @doc """
+  Update the current focus of Cortex. Arguments are (currently) the same tag filters that would be
```

Comment:
I'd drop (currently) here. Could see this getting out of date, and we might as well own it.

Reply:
sure.